models/tiltrotor/tiltrotor.sdf

```diff
@@ -1035,9 +1028,15 @@
       <accelerometerBiasCorrelationTime>300.0</accelerometerBiasCorrelationTime>
       <accelerometerTurnOnBiasSigma>0.196</accelerometerTurnOnBiasSigma>
     </plugin>
-    <plugin name='groundtruth_plugin' filename='libgazebo_groundtruth_plugin.so'>
-      <robotNamespace/>
-    </plugin>
+    <include>
```

Comment:
I would actually prefer this where it was since here it's like in the middle of the plugins.

Reply:
Done